sunpycube/cube/NDCube.py

```diff
@@ -0,0 +1,86 @@
+from sunpy.visualization.imageanimator import ImageAnimatorWCS
+import astropy.nddata
+
+__all__ = ['NDCube', 'Cube2D', 'Cube1D']
+
+
+class NDCube(astropy.nddata.NDData):
+    """docstring for NDCube"""
+
+    def __init__(self, data=None, wcs=None, **kwargs):
+        super(NDCube, self).__init__(data=data, **kwargs)
+        self.axes_wcs = wcs
+
+    def pixel_to_world(self):
+        pass
+
+    def world_to_pixel(self):
+        pass
+
+    def to_sunpy(self):
+        pass
+
+    def dimension(self):
+        pass
+
+    def plot(self, *args, **kwargs):
+        """
+        Plots an interactive visualization of this cube with a slider
+        controlling the wavelength axis.
+        Parameters other than data and wcs are passed to ImageAnimatorWCS, which in turn
+        passes them to imshow.
```

Comment:
Suggested rewrite of beginning of docstring:
```Plots an interactive visualization of this cube using sliders to move through axes plot using in the image.```
Don't need to mention the wavelength axis it it should work for any set of WCS axes.